test_scripts/API/Restructuring_OnResetTimeout/013_BC_DialNumber_with_BC_OnResetTimeout.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) DialNumber is requested
+-- 2) Some time after receiving DialNumber request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 15000) to SDL
+-- 4) HMI sends response
+-- SDL does:
+-- 1) SDL must always wait
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local paramsForRespFunction = {
+	respTime = 13000,
```

Comment:
HMI must send the response after resetPeriod is expired, so if resetPeriod = 12000, response must be equal to 14000, please update respTime = 14000